src/utils/rabbitmq.js

```diff
@@ -0,0 +1,17 @@
+const { Configuration } = require('@schul-cloud/commons');
+const amqp = require('amqplib');
+
+let connection;
+
+const createChannel = async () => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected to return a value at the end of async arrow function.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5203703)